src/ol/interaction/modifyinteraction.js

```diff
@@ -113,14 +112,21 @@ ol.interaction.Modify = function(options) {
   });
 
   /**
+   * @private
+   * @param {ol.MapBrowserEvent} mapBrowserEvent Browser event.
+   * @return {boolean} Combined condition result.
```

Comment:
It's `{boolean}`; the function returns the results from the functions invocation.